subsys/usb/Kconfig

```diff
@@ -35,6 +35,36 @@ config SYS_LOG_USB_LEVEL
 
 	- 4 DEBUG, write SYS_LOG_DBG in addition to previous levels
 
+config USB_DEVICE_VID
+	hex
+	default 0x8086 #Intel vendor ID
```

Comment:
I have kept the VID / PID, but yes there should be for each board own VID / PID, and it would be better to make in over device tree.